CodeStarterCamp_Week1/main.swift

```diff
@@ -13,73 +13,53 @@ func drawPeperoBody(body: String, topping: String) {
     print(topping + body + topping)
 }
 
-func drawPeperoBar(lenght: Int, topping: String, body: String) {
+func drawPeperoBodyAndLength(length: Int, topping: String, body: String) {
     if body == "***" {
-        for _ in 1...lenght {
+        for _ in 1...length {
             drawPeperoBody(body: body, topping: topping)
         }
     }
     else {
-        for i in 1...lenght {
-            if (i % 2) != 0{
+        for oddNumber in 1...length {
+            if (oddNumber % 2) != 0 {
                 print(" | |")
-            }
-            else {
+            } else {
                 print(body)
             }
         }
     }
 }
 
-func makePepero(lenght: Int, topping: String, body: String, barLength: Int, bar: String) {
-    drawPeperoBody(body: body, topping: topping)
-    drawPeperoBar(lenght: lenght, topping: topping, body: body)
-    drawPeperoBarLength(BarLength: barLength)
+func makePepero(lenght: Int, topping: String, body: String, sickLength: Int, bar: String) {
+    drawPeperoBodyAndLength(length: lenght, topping: topping, body: body)
+    drawPeperoBarLength(sickLength: sickLength)
 }
 
-// 실행 예시 1
-print("<정보>")
-print("길이: 10")
-print("몸통: ***")
-print("토핑:")
-print("막대길이: 4")
 
-makePepero(lenght: 10, topping: " ", body: "***", barLength: 4, bar: " | |")
+func printPeperoData (lenght: Int, body: String, topping: String, sickLength: Int){
+    print("<정보>")
+    print("길이: \(lenght)")
+    print("몸통: \(body)")
+    print("토핑: \(topping)")
+    print("막대길이: \(sickLength)")
+}
 
+// 실행 예시 1
+printPeperoData(lenght: 10, body: "***", topping:  " ", sickLength: 4)
+makePepero(lenght: 10, topping: " ", body: "***", sickLength: 4, bar: " | |")
 
 // 실행 예시 2
-print("<정보>")
-print("길이: 12")
-print("몸통: ***")
-print("토핑: &")
-print("막대길이: 4")
-
-makePepero(lenght: 12, topping: "&", body: "***", barLength: 4, bar: " | |")
+printPeperoData(lenght: 12, body: "***", topping:  "&", sickLength: 4)
+makePepero(lenght: 12, topping: "&", body: "***", sickLength: 4, bar: " | |")
 
 // 실행 예시 3
-print("<정보>")
-print("길이: 12")
-print("몸통: ***")
-print("토핑: #")
-print("막대길이: 6")
-
-makePepero(lenght: 12, topping: "#", body: "***", barLength: 6, bar:  " | |")
+printPeperoData(lenght: 12, body: "***", topping:  "#", sickLength: 6)
+makePepero(lenght: 12, topping: "#", body: "***", sickLength: 6, bar:  " | |")
```

Comment:
```suggestion
printPeperoData(lenght: 12, body: "***", topping: "#", sickLength: 6)
makePepero(lenght: 12, topping: "#", body: "***", sickLength: 6, bar: " | |")
```
요긴 공백이 하나씩 더 들어가있네요!
이런 사소한 컨벤션이 안 맞춰지면, 로직을 잘 구현했더라도 코드의 퀄리티가 낮아보일 수 있습니다 😢 
항상 커밋하기 전에 코드 컨벤션을 전체적으로 확인하는 게 좋습니다!

Reply:
머지 해주시면 감사하겠습니다 이번 요청사항으로 더 배워 다음 스텝에서 적용을 잘 해보겠습니다